sklearn/tests/test_calibration.py

```diff
@@ -166,6 +166,12 @@ def test_sample_weight(data, method, ensemble):
     X_train, y_train, sw_train = X[:n_samples], y[:n_samples], sample_weight[:n_samples]
     X_test = X[n_samples:]
 
+    scaler = StandardScaler()
```

Comment:
```suggestion
    # FIXME: ideally we should create a `Pipeline` with the `StandardScaler`
    # followed by the `LinearSVC`. However, `Pipeline` does not expose
    # `sample_weight` and it will be silently ignored.
    scaler = StandardScaler()
```